api/integrations/launch_darkly/tasks.py

```diff
@@ -0,0 +1,12 @@
+from django.db import transaction
+
+from integrations.launch_darkly.models import LaunchDarklyImportRequest
+from integrations.launch_darkly.services import process_import_request
+from task_processor.decorators import register_task_handler
+
+
+@register_task_handler()
+def process_launch_darkly_import_request(import_request_id: int) -> None:
+    with transaction.atomic():
```

Comment:
I am a bit concerned about this long-running traction: is there any way we can avoid this?

Reply:
Maybe the question is should we have it at all or should we prefer having partial success imports?

To support partial imports better, we could extend the import request status field to have a `partial_success` status and `error_messages` array. Moreover, we could have a per-object status report so users could see which exact entity failed to import.

Reply:
As discussed internally, keeping the transaction for now. Partial imports TBD in a separate PR.